code-studio/src/js/components/progress/progress_dot.jsx

```diff
@@ -163,72 +167,55 @@ export const ProgressDot = React.createClass({
     const smallDot = !this.props.courseOverviewPage && !onCurrent;
     const showLevelName = level.kind === 'named_level' && this.props.courseOverviewPage;
     const isPeerReview = level.kind === 'peer_review';
-    const isLockedReview = isPeerReview && level.status === 'not_tried';
 
-    const iconElement = (level.icon && !isPeerReview) ?
-      <i
-        className={`fa ${level.icon}`}
-        style={[
-              styles.dot.common,
-              styles.dot.puzzle,
-              this.props.courseOverviewPage && styles.dot.overview,
-              styles.dot.icon,
-              smallDot && styles.dot.icon_small,
-              level.status && level.status !== 'not_tried' && styles.dot.icon_complete,
-              outlineCurrent && {textShadow: createOutline(color.level_current)}
-            ]}
-        /> :
-      <div
-        className={`level-${level.id} ${isPeerReview && `fa ${level.icon}`}`}
-        style={[
-              styles.dot.common,
-              isLockedReview ? styles.dot.lockedReview : styles.dot.puzzle,
-              this.props.courseOverviewPage && styles.dot.overview,
-              smallDot && styles.dot.small,
-              level.kind === 'assessment' && styles.dot.assessment,
-              outlineCurrent && {borderColor: color.level_current},
-              showUnplugged && styles.dot.unplugged,
-              styles.status[level.status || 'not_tried'],
-            ]}
-        >
-        {(showLevelName || (isPeerReview && level.icon === '')) ? '\u00a0' : level.title}
-      </div>;
-
-    const rowText = (showLevelName || isPeerReview) &&
-      <span
-        key='named_level'
-        style={[styles.levelName, isLockedReview && {color: color.charcoal}]}
+    return (
+      <a
+        key='link'
+        href={level.locked ? undefined : level.url}
+        onClick={this.props.saveAnswersBeforeNavigation && dotClicked.bind(null, level.url)}
+        style={[styles.outer, (showLevelName || isPeerReview) && {display: 'table-row'}, level.locked && styles.disabledLevel]}
       >
-        {level.name}
-      </span>;
-
-
-    if (isLockedReview) {
-      return (
+        {(level.icon && !isPeerReview) ?
+        <i
+          className={`fa ${level.icon}`}
+          style={[
+            styles.dot.common,
+            styles.dot.puzzle,
+            this.props.courseOverviewPage && styles.dot.overview,
+            styles.dot.icon,
+            smallDot && styles.dot.icon_small,
+            level.status && level.status !== 'not_tried' && styles.dot.icon_complete,
+            outlineCurrent && {textShadow: createOutline(color.level_current)}
+          ]}
+        /> :
         <div
-          key='lockedReview'
-          style={[styles.outer, {display: 'table-row'}]}
+          className={`level-${level.id} ${isPeerReview && `fa ${level.icon}`}`}
```

Comment:
I don't see how this goes away - it's needed to be fa-check for a completed review, fa-lock for a locked row, and nothing in other cases.